app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
-  belongs_to :user
   belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to the database, then make the `update_comments_counter` method private.